File: cw/cpp/string_repeat.cpp
// https://www.codewars.com/kata/57a0e5c372292dd76d000d7e/train/cpp


#include <iostream>

std::string repeat_str(size_t repeat, const std::string& str) {
    std::string repeated;

    for(int i = 0; i < repeat; ++i) {
        repeated += str;
    }

    return repeated;
}

int main() {
    size_t repeat;
    std::cin >> repeat;

    std::string str;
    std::cin >> str;

    std::cout << repeat_str(repeat, str) << "\n";

    return 0;
}

File: codewars/elixir/string_repeat.exs
# https://www.codewars.com/kata/57a0e5c372292dd76d000d7e/train/elixir

defmodule Solution do
  def repeat_str(n, s) do
    1..n
      |> Enum.map(fn(_) -> s end)
      |> Enum.join()
  end
end

IO.inspect(Solution.repeat_str(12, "Bee"))
